recipes/tptp-lark-parser/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "tptp-lark-parser" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tptp-lark-parser-{{ version }}.tar.gz
+  sha256: f54129f19af4d34003d08538823cb0190611387affa7df0b55723fa0cb55163f
+
+build:
+  skip: true  # [py>=311 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry-core >=1.0.0
+  run:
+    - importlib_resources  # [py<39]
```

It is fine to install `importlib_resources` in >=39, the standard lib will be used there and we can make the package noarch.

```suggestion
    - importlib_resources
```